src/frontend/src/lib/components/signer/SignerOrigin.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+	import type { Origin, PayloadOrigin } from '@dfinity/oisy-wallet-signer';
+	import { isNullish, nonNullish } from '@dfinity/utils';
+	import ExternalLink from '$lib/components/ui/ExternalLink.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { Option } from '$lib/types/utils';
+
+	export let payload: Option<PayloadOrigin>;
+
+	let origin: Origin | undefined;
+	$: origin = payload?.origin;
+
+	const mapHost = (origin: Origin | undefined): Option<string> => {
+		if (isNullish(origin)) {
+			return undefined;
+		}
+
+		try {
+			// If set we are actually sure that the $payload.origin is a valid URL, thanks to the library but, for the state of the art, we still catch potential errors here too.
+			const { host } = new URL(origin);
+			return host;
+		} catch {
+			return null;
+		}
+	};
+
+	// Null being used if mapping the origin does not work - i.e. invalid origin. Probably an edge case.
+	// eslint-disable-next-line local-rules/use-option-type-wrapper
+	let host: string | null | undefined;
+	$: host = mapHost(origin);
+</script>
+
+{#if nonNullish(origin)}
+	<p class="mb-6 break-normal text-center">
+		{$i18n.signer.origin.text.request_from}
+		{#if nonNullish(host)}<span class="font-bold text-brandeis-blue"
```

Comment:
is this the blue that was decided officially? because otherwise the prop color='blue' of `ExternalLink` should provide the other "official" blue

Reply:
Just implemented what was provided in the Figma file I got 🤷‍♂️